cmake/modules/AddSwift.cmake

```diff
@@ -997,7 +1021,7 @@ function(_add_swift_library_single target name)
     set_target_properties("${target}"
       PROPERTIES
       INSTALL_NAME_DIR "${install_name_dir}")
-  elseif("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
+  elseif("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux" AND NOT "${SWIFTLIB_SINGLE_SDK}" STREQUAL "ANDROID")
```

Comment:
Interesting.  How do apps typically bundle libraries?  Do they have to call `System.loadLibrary()`?  Is that the recommended approach?

I'm just concerned about putting in a hack that would cause issues later.  For example, how would an app with a Swift main() work?  It won't be able to call `loadLibrary()` before running Swift code.